src/code/z_actor.c

```diff
@@ -2614,200 +2614,128 @@ void Actor_DrawLensOverlay(Gfx** gfxP, s32 lensMaskSize) {
                                        ((LENS_MASK_ACTIVE_SIZE - lensMaskSize) * 0.003f) + 1.0f);
 }
 
-#ifdef NON_EQUIVALENT
-void Actor_DrawLensActors(PlayState* play, s32 numActors, Actor** actors) {
+void Actor_DrawLensActors(PlayState* play, s32 numInvisibleActors, Actor** invisibleActors) {
+    s32 dbgVar1;
+    s32 dbgVar2;
     s32 spB4;
+    Actor** invisibleActor;
     Gfx* spAC;
-    void* spA8; // pad
+    void* spA8;
     s32 spA4;
```

Comment:
Would be better to type `spA4` as a `UNK_PTR` instead?

Reply:
looking at the play PR, `unk_18E68` seems to be a `void*`, so probably just `void*` should be fine?

Reply:
It should be the same type, at least.

Reply:
yeah, `void*` sounds fine